dev-manual/api/api-reference-archivematica.rst

```diff
@@ -484,13 +484,45 @@ Response definitions:
 
 ==================   ==========================================================
 
-``message``          "Approval successful."
+``error``            Boolean true or false.
 
-``reingest_uuid``    UUID of the reingested transfer.
+``message``          Success message or reason for the error.
 
 ==================   ==========================================================
 
 
+Usage example::
+
+  curl -X POST \
+  http://my_archivematica_instance.archivematica.org/api/ingest/copy_metadata_files/ \
+  -H 'Authorization: ApiKey [_your_username_]:[_your_ApiKey_]' \
+  -H 'Content-Type: application/x-www-form-urlencoded' \
+  -d 'sip_uuid=a7f1f36c-55c1-4fb7-a898-76d102e847dc&source_paths%5B%5D='''NmRjYTgxNDAtMDg2NC00MzE5LTg2ZDctNTg0ZjZiZTY4N2EzOi9ob21lL2FyY2hpdmVtYXRpY2EvYXJjaGl2ZW1hdGljYS1zYW1wbGVkYXRhL2lzc3VlX3RlbXBsYXRlLm1k''''
+
+
+Response examples
+
+200 OK::
+
+    {
+      "error": false,
+      "message": "Metadata files added successfully."
+    }
+
+400 Bad Request ::
+
+    {
+      "error": true,
+      "message": "sip_uuid and source_paths[] both required."
+    }
+
+500 Internal Server Error::
+
+    {
+      "error": true,
+      "message": "sip_uuid and source_paths[] both required."
```

Comment:
Reading [the code](https://github.com/artefactual/archivematica/blob/d42c61206b0e82af9f6a39386b801c2d39a0e827/src/dashboard/src/components/filesystem_ajax/views.py#L963-L970) I don't think this error message is attached to the `500` error.

```suggestion
      "message": "Location 6dca8140-0864-4319-86d7-584f6be687a3 is not associated with this pipeline"
```

Reply:
Let me go through more exceptions.